### DISCUSSION TITLE
エラー

### DISCUSSION THREAD

**Discussion by 11EDA** [aclap-dev/video-downloadhelper#544] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://missav.com/ja/luxu-932-uncensored-leak",
  "error": {
    "details": "SideDownload error: : [in#0 @ 000001956261b800] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://surrit.com/9d73dac2-ab45-4446-a661-f55b3425e2ee/360p/video.m3u8.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "downloadable_id": "downloadable_784037876430779",
    "error": "coapp_failure",
    "id": "coapp_failure_ad7a4915-659b-4655-80cd-54eff3a8dc79"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: google
channel: stable
lang: ja
coapp: {"found":true,"new_version":false,"path":"D:\\Downloads\\DL\\DownloadHelper\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"0414f58ff7cb4bd1"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "D:\\Downloads\\DL",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 7
}

**Comment by GrampaWildWilly** [comment]
Read #519.  Be sure you click any links that say, "Show x previous replies."
